index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Hello Microverse</title>
+    <link rel="stylesheet" href="C:/HelloMicroverse/styles.css" />
+  </head>
+  <body>
+    <h1>Hello Microverse!</h1>
+  </body>
```

Comment:
Great job @itsmraga-hub. ✔️

- [ ] Please try to link your style with your HTML file properly. Otherwise, the styles are not reflecting well on the HTML page.

Reply:
Do I have to create another Pull request or do I use this same one to make the changes?

Reply:
@1GORDON First thank you for the feedback.